### ISSUE TITLE
Update Axe-core and fix any errors

### ISSUE THREAD

**Issue by evgenitsn** [newscorp-ghfb/newskit#168]
### Description

Dow Jones reported an A11Y error coming from the Tab component (having a quick check with Stoyan the response seems to actually be the scroll)Render reported also an issue where their select input for Country (on address form) was failing when running the axe-dev plugin.The select is receiving an [aria-described by id for an element not present in the DOM](https://dequeuniversity.com/rules/axe/4.4/aria-valid-attr-value?application=AxeChrome)[](https://newsuktechnology.slack.com/archives/C024ZGQ2W8K/p1649259619247679)

Notify render team if this gets fixed

We haven’t had any test failing with that error, even if we run Axe-core. This because our version is a bit old (29/10/2021) and they have added a new set of rules in the last months.

We should then update axe-core

Also update cypress-axe

Fix any issues (timebox to 3 days) if we don’t fix all the issue create a new ticket/s with next steps

Then notify DOW once fixed and on the next release

Notify Render team once fixed and on the next release

**ACCEPTANCE CRITERIA**

Axe-core should be updated to the latest version
We should not have any errorNotify DOW that their issue has been solved, hopefully should come up as  an error once we update the package.

**Timebox**

3 days
